versions.props

```diff
@@ -18,6 +18,7 @@ org.apache.arrow:arrow-memory-netty = 2.0.0
 com.github.stephenc.findbugs:findbugs-annotations = 1.3.9-1
 software.amazon.awssdk:* = 2.15.7
 org.scala-lang:scala-library = 2.12.10
+org.scala-lang.modules:scala-java8-compat_2.12 = 0.8.0
```

Comment:
I think we can just use Optional.apply() instead of using the option converter here

Reply:
It would be nice to not have an additional explicit scala 2.12 dependency. I believe Spark2 can still be compiled for scala 2.11. Would this prohibit that?

Reply:
Yep you guys convinced me, got rid of it.